frontend/src/components/layout/header/TopNavbar.vue

```diff
@@ -11,33 +11,14 @@
             </span>
             <span>Metrica</span>
         </VToolbarTitle>
-        <UserControls v-if="isLoggedIn" />
```

Comment:
It will remove user info from header completely, not sure that we need to do this

Reply:
Oh my Lord! You are right! Returned.